src/main/kotlin/lotto/domain/Prize.kt

```diff
@@ -1,24 +1,16 @@
 package lotto.domain
 
-enum class Prize(val countOfMatch: Int, private val rank: Int, val prizeMoney: Int, var count: Int = 0) {
-    MATCH_NONE(0, 0, 0),
-    MATCH_ONE(1, 0, 0),
-    MATCH_TWO(2, 0, 0),
+enum class Prize(val countOfMatch: Int, private val rank: Int, val prizeMoney: Int, val withBonus: Boolean = false) {
     MATCH_THREE(3, 5, 5_000),
```

Comment:
상수 이름은 `FIFTH`가 적절해 보여요.

```suggestion
    FIFTH(3, 5, 5_000),
```